src/constants.json

```diff
@@ -65,23 +65,23 @@
     "FORMAT": "hb_format"
   },
   "NATIVE_KEYS": {
-    "title": "hb_native_title",
-    "body": "hb_native_body",
-    "body2": "hb_native_body2",
-    "privacyLink": "hb_native_privacy",
-    "sponsoredBy": "hb_native_brand",
-    "image": "hb_native_image",
-    "icon": "hb_native_icon",
-    "clickUrl": "hb_native_linkurl",
-    "displayUrl": "hb_native_displayurl",
-    "cta": "hb_native_cta",
-    "rating": "hb_native_rating",
-    "address": "hb_native_address",
-    "downloads": "hb_native_downloads",
-    "likes": "hb_native_likes",
-    "phone": "hb_native_phone",
-    "price": "hb_native_price",
-    "salePrice": "hb_native_saleprice"
+    "title": "pwt_native_title",
+    "body": "pwt_native_body",
+    "body2": "pwt_native_body2",
+    "privacyLink": "pwt_native_privacy",
+    "sponsoredBy": "pwt_native_brand",
+    "image": "pwt_native_image",
+    "icon": "pwt_native_icon",
+    "clickUrl": "pwt_native_linkurl",
+    "displayUrl": "pwt_native_displayurl",
+    "cta": "pwt_native_cta",
+    "rating": "pwt_native_rating",
+    "address": "pwt_native_address",
+    "downloads": "pwt_native_downloads",
+    "likes": "pwt_native_likes",
+    "phone": "pwt_native_phone",
+    "price": "pwt_native_price",
```

Comment:
what are the purpose of  pwt_native_price and pwt_native_saleprice keys?
how are we going to send pwtecp key?

Reply:
I checked and both price and saleprice are data type asset for native creative 
https://iabtechlab.com/wp-content/uploads/2016/07/OpenRTB-Native-Ads-Specification-Final-1.2.pdf